annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -369,6 +369,7 @@ private String resolveHgvsp() {
     }
 
     private String resolveHgvspShort() {
+        String variantClassification = resolveVariantClassification();
```

Comment:
You don't use this until the nested else if (line 396) - I suggest you move it down to where it is used.